forge-gui/res/cardsfolder/upcoming/blaster_combat_dj_blaster_morale_booster.txt

```diff
@@ -0,0 +1,26 @@
+Name:Blaster, Combat DJ
+ManaCost:3 R G
+Types:Legendary Artifact Creature Robot
+PT:3/3
+K:More Than Meets the Eye:1 R G
+S:Mode$ Continuous | Affected$ Artifact.Creature+Other+YouCtrl+nonToken,Vehicles.Other+YouCtrl+nonToken | AddKeyword$ Modular:1 | Description$ Other nontoken artifact creatures and Vehicles you control have modular 1. (They enter the battlefield with an additional +1/+1 counter on them. When they die, you may put their +1/+1 counters on target artifact creature.)
+T:Mode$ CounterAddedOnce | CounterType$ P1P1 | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigConvert | TriggerDescription$ Whenever you put one or more +1/+1 counters on NICKNAME, convert it.
+SVar:TrigConvert:DB$ SetState | Mode$ Convert
+AlternateMode:Convert
+DeckHas:Ability$Counters & Keyword$Modular
+DeckHints:Type$Vehicle|Artifact
+Oracle:More Than Meets the Eye {1}{R}{G} (You may cast this card converted for {1}{R}{G}.)\nOther nontoken artifact creatures and Vehicles you control have modular 1. (They enter the battlefield with an additional +1/+1 counter on them. When they die, you may put their +1/+1 counters on target artifact creature.)\nWhenever you put one or more +1/+1 counters on Blaster, convert it.
+
+ALTERNATE
+
+Name:Blaster, Morale Booster
+ManaCost:no cost
+Colors:red,green
+Types:Legendary Artifact
+K:Modular:3
+A:AB$ MoveCounter | Cost$ X T | Source$ Self | ValidTgts$ Artifact.Other| TgtPrompt$ Select target artifact to get counters | CounterType$ P1P1 | CounterNum$ X | SorcerySpeed$ True | SubAbility$ DBAnimate | SpellDescription$ Move X +1/+1 counters from NICKNAME onto another target artifact. That artifact gains haste until end of turn. If Blaster has no +1/+1 counters on it, convert it. Activate only as a sorcery.
```

Comment:
- [ ] spacing
- [ ] TgtPrompt$ should include "other"
- [ ] split this SpellDescription as appropriate to lines 22 and 23 – some of those lines may need a StackDescription too (probably line 23)